src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -155,39 +142,68 @@ namespace DataHeatBalance {
     constexpr int DefaultMinNumberOfWarmupDays(1);  // Default minimum number of warmup days allowed
 
     // Parameters for Sky Radiance Distribution
-    constexpr int Isotropic(0);
-    constexpr int Anisotropic(1);
+    enum class SkyRadDistribution
+    {
+        Unassigned = -1,
+        Isotropic,
+        Anisotropic
+    };
 
     // Parameters for ZoneAirSolutionAlgo
-    constexpr int Use3rdOrder(0);
-    constexpr int UseAnalyticalSolution(1);
-    constexpr int UseEulerMethod(2);
+    enum class SolutionAlgo
+    {
+        Unassigned = -1,
+        ThirdOrder,
+        AnalyticalSolution,
+        EulerMethod
+    };
 
     // Parameter for MRT calculation type
-    constexpr int ZoneAveraged(1);
-    constexpr int SurfaceWeighted(2);
-    constexpr int AngleFactor(3);
+    enum class CalcMRT
+    {
+        Unassigned = -1,
+        ZoneAveraged,
+        SurfaceWeighted,
+        AngleFactor
+    };
```

Comment:
You're doing a ```static_cast<int>(mrtCalcType)``` in SQLiteProcedures.cc which uses these enum values. It looks like you aren't actually required to populate all of enum values for this to work as expected (see [Compiler Explorer example](https://godbolt.org/z/ojY7r9566)). However, do we want to explicitly fill them in anyways? 

@Myoldmopar @amirroth

Reply:
I think that comes down to style.  If you explicitly list out the values, then you definitely know what they are, but then on longer lists, you are now adding something to maintain, rather than just letting the compiler assign them values internally.  I'm not getting a strong instinct one way or the other.

Reply:
I'm inclined to leave it as-is and let the compiler figure it out. One potential issue with explicitly listing them out is that C++ is happy to let you have (by mistake or otherwise) the following:

```C++
enum class MyEnum
{
   Invalid = -1,
   A = 1,
   B = 1,
}
```

Then ```MyEnum::A == MyEnum::B``` evaluates to True. See another [Compiler Explorer example](https://godbolt.org/z/43dPzeonx). This seems to me like it's going to be prone to errors.

One more question: do we need to inherit from ```int``` on all of these? i.e. ```enum class MyEnum : int {}```. It seems the answer is "no", but I just want to double-check.

Reply:
It defaults to int, so if we are going with int then we shouldn't need that listed anywhere.  I wonder if we should consider using `char`?    That's for a different day...

> Each enumeration also has an underlying type. The underlying type can be explicitly specified using an enum-base. For a scoped enumeration type, the underlying type is int if it is not explicitly specified. In both of these cases, the underlying 
 type is said to be fixed. 

[_C++ 20 Standard_](https://isocpp.org/files/papers/N4860.pdf)